### ISSUE TITLE
Why are size stats units given as thousands?

### ISSUE THREAD

**Issue by fracai** [gilbertchen/duplicacy#435]
This is a relatively minor point, but why are the size stats (duplicacy check -stats) given in thousands of units? As in, 1 GB of data is presented as 1,000M. 1 MB is presented as 1,000K. Even knowing what is being shown, I still need to make two conversions in my head. First, pretend that the comma is really a decimal. Then, increase the units notation.

The fix would be a small change to the `PrettyNumber` and `PrettySize` methods in `duplicacy_utils.go`, but I figure there's probably a reason for the current appearance. Would this be a welcome pull request or is the current behavior explicitly preferred?

**Comment by gilbertchen** [comment]
It was a mistake to use 1000 instead of 1024.  Please create a pull request if you can.

**Comment by fracai** [comment]
To clarify, it's no so much 1000 vs 1024 that I was talking about (though I can fix that too).
I was talking about how sizes are reported like:
`123,456 M`
instead of:
`123.456 G`
or, with the 1000 to 1024 fix:
`120.563 G`

It's mostly the units that I'm questioning.

**Comment by NiJoao** [comment]
I agree that 3 places before and after a comma is clearer, don't need to go into the thousands units.
But please, please, don't use MB/GB in powers of 1024. Always use SI units, MB with 1000 or MiB with 1024, https://en.m.wikipedia.org/wiki/Mebibyte

**Comment by fracai** [comment]
I created a pull request that illustrates what I'm going for.
It also uses the SI units.
